packages/consumption/src/modules/attributes/AttributesController.ts

```diff
@@ -1049,6 +1081,46 @@ export class AttributesController extends ConsumptionBaseController {
         }
     }
 
+    private async transmitDefault(attribute: LocalAttribute): Promise<void> {
+        if (!attribute.default) return;
+
+        const valueType = attribute.content.value.constructor.name as AttributeValues.Identity.TypeName;
+        const query: IIdentityAttributeQuery = {
+            valueType: valueType
+        };
+
+        const attributesWithSameValueType = await this.executeIdentityAttributeQuery(query);
+        if (attributesWithSameValueType.length <= 1) return;
```

Comment:
I think it would be better to adjust the `query` used (for the case of Attribute succession). If you previously had a RepositoryAttribute with predecessor, `attributesWithSameValueType.length` is automatically greater than one, right? (Even though there is no RepositoryAttribute left as an appropriate default RepositoryAttribute.)

Reply:
Since we delete the predecessors before transmitting the default, this shouldn't pose a problem. I added a test for it.

Reply:
Good point. However, would still adapt the `query` to increase the reusability of the function and reduce the error-proneness of the function, so that the functionality of the function is not so strongly influenced by its position within a function call.